tests/pems/crl/generate.sh

```diff
@@ -0,0 +1,203 @@
+#!/bin/bash
+
+get_subj() {
+  common_name="$1"
+  echo "/C=US/ST=Massachusetts/L=Boston/CN=${common_name}"
+}
```

Comment:
I always forget exactly how this works, but make sure your certs will pass s2n-tls's [default verify_host callback](https://github.com/aws/s2n-tls/blob/c42717c6a52ab9b81fd11e0ebcc52fd2dcbb8eb3/tls/s2n_connection.c#L291-L323) in the tests. If I remember right, that means the CN has to be localhost? Maybe check the existing certs.

Reply:
I think you can check whether it'll pass by using s2nc/s2nd WITHOUT validation disabled (so no -i).

Reply:
I checked some exiting certs, and you were correct, the CN has to be localhost. I added that here: https://github.com/aws/s2n-tls/blob/c1615a6fcf518ac9d84564d9647d97029f3106b2/tests/pems/crl/generate.sh#L47-L50

I also [created a new script](https://github.com/aws/s2n-tls/blob/c1615a6fcf518ac9d84564d9647d97029f3106b2/tests/pems/crl/test_s2nc.sh) to test all of the certificates with s2nc/s2nd, and the generated certs now pass this test.